### ISSUE TITLE
Add reasonable colors for pre-processors like SCSS

### ISSUE THREAD

**Issue by davidsneighbour** [matthewhowell/reasonable-colors#1]
It's currently not possible to use `var(--color)` in SASS/SCSS files. A good feature would be to ass something along the lines of `reasonable-colors.scss` with content like:

```
  $color-red-1: rgb(255, 248, 246);
  $color-red-2: rgb(255, 221, 216);
  $color-red-3: rgb(255, 70, 71);
  $color-red-4: rgb(224, 0, 43);
  $color-red-5: rgb(131, 0, 20);
  $color-red-6: rgb(83, 0, 3);
```

no :root-brackets required for that.

**Comment by matthewhowell** [comment]
Thanks for this! Restating the issue: the goal is to make the color variables available for use with CSS preprocessors, specifically SCSS and SASS.

I think (please correct if I'm wrong) that SCSS compatibility is simpler, since valid SCSS is valid CSS. For this, just offering a .scss files with the variables no longer indented under the :root pseudo-class (as you stated) will solve the issue.

And for SASS, removing the semi-colons and indentation would be sufficient? Perhaps, since converting between SCSS and SASS is rather simple, I'd leave this out of the repo? Figuring that providing the SCSS files would be sufficient. Thoughts?

If this sounds good, I'll add new source files for .scss for each color space. So we'd end up with:
- reasonable-colors.css
- reasonable-colors.scss
- reasonable-colors-hsl.css
- reasonable-colors-hsl.scss
- reasonable-colors-lch.css
- reasonable-colors-lch.scss
- reasonable-colors-rgb.css
- reasonable-colors-rgb.scss

**Comment by ianhobbs** [comment]
Yes this is a great idea. 
I'd go one step further and suggest that the authoring engine of these scales cold be adapted the to the repo so that you can generate your own 'colours'. 
Create the 5 tone set and/or the triadic etc. Eg. 

https://sass-lang.com/documentation/modules/color

Example : generate complement colour.

// Hue 222deg becomes 42deg.

color.complement(#6b717f)  // #7f796b

**Comment by matthewhowell** [comment]
I like this idea as well. At the moment, I'm not sure this stuff should live in the colors repo, or if it's out of scope and needs to belong in a palettes file (or another repo). I'll have to think on it.

Would you be interested in this mainly as a SASS module? Or just generally you'd like to be able to quickly use these relationships in your .css or .scss files?

**Comment by matthewhowell** [comment]
Closing this issue, addressed by 0.4.0 which added .scss files for each color space.
